packages/ui_v2/src/components/commonV2/dao/Migrator/DeployButton.tsx

```diff
@@ -417,19 +434,28 @@ export const DeployButton: FC<IProps> = ({ migrationStates }) => {
     if (!result) return;
 
     let network = await getNetworkName();
-    if (network === 'private') {
-      if (await web3.eth.net.getId() === 100) {
-        network = 'xdai'
-      } else if (await web3.eth.net.getId() === 77) {
-        network = 'sokol'
+    if (network === "private") {
+      if ((await web3.eth.net.getId()) === 100) {
+        network = "xdai";
+      } else if ((await web3.eth.net.getId()) === 77) {
+        network = "sokol";
       }
     }
 
     let url;
-    if (network === "mainnet") url = `https://alchemy.daostack.io/dao/${result.Avatar}`;
-    else if (network === "rinkeby") url = `https://alchemy-staging-rinkeby.herokuapp.com/dao/${result.Avatar}`;
-    else if (network === "xdai") url = `https://alchemy-2-xdai.herokuapp.com/dao/${result.Avatar}`;
-    else url = `/dao/${result.Avatar}`;
+    if (network === "mainnet")
+      url =
+        `${redirectURL}/dao/${result.Avatar}` ||
```

Comment:
Does this actually work? If `redirectURL` is `undefined`, is the whole string `undefined`?

Reply:
Awesome catch, ty!